templates/note.php

```diff
@@ -1,3 +1,4 @@
 <div id="app-navigation-toggle" class="icon-menu" style="display:none;"></div>
 	<pre editor notes-timeout-change="save()" notes-autofocus ng-class="{saving: isSaving()}">{{note.content}}</pre>
+	<div class="note-meta">{{note.content | wordCount }} words</div>
```

Comment:
@jancborchardt Maybe like this?
https://github.com/owncloud/notes/commit/e993536443a13435e567218007115a6425e459b4